AlphaWallet/UI/Views/AmountTextField.swift

```diff
@@ -145,50 +148,105 @@ class AmountTextField: UIControl {
         return label
     }()
 
+    private var cryptoToDollarRatePrevValue: Double?
```

Comment:
Can you comment on what this new set of properties do and help, maybe with respect to the 2 issues? These few classes are still quite fragile so I'm hoping we can make minimal changes to it.